opensquirrel/default_measurements.py

```diff
@@ -1,14 +1,14 @@
-from opensquirrel.ir import Measure, Qubit, named_measurement
+from opensquirrel.ir import Bit, Measure, Qubit, named_measurement
 
 
 @named_measurement
-def measure(q: Qubit) -> Measure:
-    return Measure(qubit=q, axis=(0, 0, 1))
+def measure(b: Bit, q: Qubit) -> Measure:
+    return Measure(bit=b, qubit=q, axis=(0, 0, 1))
```

Comment:
I think we should follow the Qiskit convention of the order of the parameters, _i.e._, first the qubit that is measure and then the bit into which the measurement outcome is stored: see https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit#measure

Reply:
I agree. I did it this way because the semantic AST has a bit first operand and a qubit second operand. But that's not strong enough a reason to keep that order in OpenSquirrel. I'll put bits behind. Thanks!

Reply:
Done!